Xamarin.Forms.Core/Device.cs

```diff
@@ -164,6 +164,22 @@ public static async Task<SynchronizationContext> GetMainThreadSynchronizationCon
 			return ret;
 		}
 
+		static Page GetElementPage(Element element)
+		{
+			if (element == null)
+			{
+				return null;
+			}
+			else if (element is Page page)
+			{
+				return page;
+			}
+			else
+			{
+				return GetElementPage(element.Parent);
```

Comment:
can we replace the recursion by a while loop ?

Reply:
I can do you one better - we can replace this with no method at all, since it's not used anymore.

Reply:
I remember removing this method ... I don't know what happened ...